Scripts/DEADataHandling.py

```diff
@@ -110,22 +109,10 @@ def load_nbarx(dc, sensor, query, product='nbart', bands_of_interest='', filter_
             ds.attrs['crs'] = crs
             ds.attrs['affine'] = affine
 
-        # If product is NBAR-T, also correct terrain by replacing -999.0 with nan
-        if product == 'nbart':
+        # Replace nodata values with nans
 
-            print('Masked {} with {} and filtered terrain'.format(product_name,
-                                                                  mask_product))
             ds = ds.where(ds != -999.0)
```

Comment:
Hard-coding this value isn't the best. The value is stored as a attr on the data array.
Check out the `mask_invalid_data` functions here: http://datacube-core.readthedocs.io/en/latest/dev/api/masking.html